src/panels/config/script/ha-script-editor.ts

```diff
@@ -894,6 +922,13 @@ export class HaScriptEditor extends SubscribeMixin(
     }
     const id = this.scriptId || this._entityId || Date.now();
 
+    await this._saveScript(id);
+    if (!this.scriptId) {
+      navigate(`/config/automation/edit/${id}`, { replace: true });
```

Comment:
This should point to the scripting url, not the automation.

Reply:
Dang it, true, well spotted 👍